handlers/meshery_filter_handler.go

```diff
@@ -18,6 +18,14 @@ type MesheryFilterRequestBody struct {
 	FilterData *models.MesheryFilter `json:"filter_data,omitempty"`
 }
 
+// swagger:route GET /api/experimental/filter/file/{id} FiltersAPI idGetFilterFiles
+// Handle GET request for filter file with given id
+//
+// Returns the Meshery Filter file saved by the current user with the given id
+// This will return the Meshery Filter file with their details of the given id
```

Comment:
comment repeated x2

Reply:
This should be changed to `Meshery Filters file` right ?

Reply:
I mean why the same line is written twice, just a bit differently!

Reply:
Oh okay I got it. I changed it accordingly.